docs/web3.main.rst

```diff
@@ -89,23 +89,23 @@ Encoding and Decoding Helpers
 
 .. _JSON-RPC spec: https://github.com/ethereum/wiki/wiki/JSON-RPC#hex-value-encoding
 
-.. py:method:: Web3.toText(primitive=None, hexstr=None, text=None)
```

Comment:
same comment here as in the last about leaving the `toText` section, and noting that it's deprecated.

Reply:
done!